templates/enmasse-operator/050-Deployment-enmasse-operator.yaml

```diff
@@ -73,6 +73,8 @@ spec:
           value: ${KEYCLOAK_IMAGE}
         - name: KEYCLOAK_PLUGIN_IMAGE
           value: ${KEYCLOAK_PLUGIN_IMAGE}
+        - name: CONTROLLER_ENABLE_CONSOLE_SERVICE
+          value: "true"
```

Comment:
What happens if you set this to false? Will the per-addressspace console still function?

Reply:
No, the addresspace console will require OAuth too.   I think it would be best if on OpenShift, the route to the console didn't get created if the console service is absent.